SM/internal/services/shiftConnection.go

```diff
@@ -0,0 +1,45 @@
+package services
+
+import (
+	"context"
+	"sm/internal/database/postgres"
+	"sm/internal/utils/logger"
+)
+
+type ShiftWorkersListToTransfer struct {
+	Valid          bool
+	WorkersListDTO []ShiftWorkerDTO
+}
+
+type ShiftTasksListToTransfer struct {
```

Comment:
hm. is it similar to DTO, why not to be consistent and keep the same DTO postfix for each model?